### DISCUSSION TITLE
Ctrl + D to close the terminal window

### DISCUSSION THREAD

**Discussion by rrrnld** [kovidgoyal/kitty#5535] [Q&A] [answered]
Hi! I'm using kitty with fish shell on wayland. I used alacritty before, where pressing `Ctrl + D` on an empty command prompt closes the window. With kitty it just seems to exit the shell? I can't type anything anymore, but the window stays open. Is there any way to configure this?

My config looks like this (on NixOS):

```
# Generated by Home Manager.
# See https://sw.kovidgoyal.net/kitty/conf.html

include /nix/store/hjhn3ja9g9anr225011x48lkgr192624-kitty-themes-unstable-2022-08-11/themes/Jellybeans.conf

background_opacity 0.96
bold_font auto
bold_italic_font auto
font_family Hasklug Nerd Font Mono
font_size 11
hide_window_decorations yes
initial_window_height 600
initial_window_width 800
italic_font auto
shell_integration enabled
wheel_scroll_multiplier 10.0
window_margin_width 6
```

**Comment by kovidgoyal** [comment]
See https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.confirm_os_window_close

**Comment by kovidgoyal** [comment, marked as answer]
Sorry wrong link: https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.close_on_child_death

**Reply by rrrnld** [reply]
Fantastic, thank you!

**Reply by IrrerPolterer** [reply]
Exactly what I was looking for too!
